workitem/workitem_repository_blackbox_test.go

```diff
@@ -86,6 +93,20 @@ func (s *workItemRepoBlackBoxTest) TestLoadID() {
 }
 
 func (s *workItemRepoBlackBoxTest) TestCreate() {
+	s.T().Run("create work item without assignees & labels", func(t *testing.T) {
+		fxt := tf.NewTestFixture(t, s.DB, tf.WorkItemTypes(1))
+		wi, err := s.repo.Create(
+			s.Ctx, fxt.Spaces[0].ID, fxt.WorkItemTypes[0].ID,
+			map[string]interface{}{
+				workitem.SystemTitle: "some title",
+				workitem.SystemState: workitem.SystemStateNew,
+			}, fxt.Identities[0].ID)
```

Comment:
Do this instead:

```go
fxt := tf.NewTestFixture(t, s.DB, tf.WorkItems(1, func(fxt tf.TestFixture, idx int) error {
    fxt.WorkItems[idx].Fields[workitem.SystemAssignees] = nil
    fxt.WorkItems[idx].Fields[workitem.SystemLabels] = nil
    return nil
})
```

Reply:
It is not nil, rather the key itself will not be present.

Reply:
Then just do: 
```go
fxt := tf.NewTestFixture(t, s.DB, tf.WorkItems(1, func(fxt tf.TestFixture, idx int) error {
    fxt.WorkItems[idx].Fields = map[string]interface{
        workitem.SystemTitle: "some title",
        workitem.SystemState: workitem.SystemStateNew,
    }
    return nil
})
```

Reply:
I can use this in the above `TestSave`, but in this particular case, the function under test is `Create`.